cmake/thirdparty/SetupSeracThirdParty.cmake

```diff
@@ -221,6 +221,15 @@ if (NOT SERAC_THIRD_PARTY_LIBRARIES_FOUND)
             set(SuperLUDist_DIR ${SUPERLUDIST_DIR} CACHE PATH "")
             set(MFEM_USE_SUPERLU ${ENABLE_MPI} CACHE BOOL "")
         endif()
+        if(STRUMPACK_DIR)
+            serac_assert_is_directory(VARIABLE_NAME STRUMPACK_DIR)
+            set(MFEM_USE_STRUMPACK ON CACHE BOOL "")
+            find_package(strumpack CONFIG PATHS ${STRUMPACK_DIR}/lib/cmake/STRUMPACK)
```

Comment:
I've forgotten, did we end up confirming we needed this `find_package`?

Reply:
unsure - I can check

Reply:
yeah, we need it.